### ISSUE TITLE
AttributeError: 'Manager' object has no attribute '_prefetch_related_lookups'

### ISSUE THREAD

**Issue by guglielmo** [taigaio/taiga-back#603]
I'm following the instructions at http://taigaio.github.io/taiga-doc/dist/setup-production.html.

After installing taiga-back, I receive this error when visiting the `/api/v1/projects` endpoint (and many others in `/api/v1/`)

I am using the `stable` branch, installed as per the instructions on a Ubuntu 14.04 distro, as `taiga` user.

```
[05/Feb/2016 11:33:31] "GET /api/v1/ HTTP/1.1" 200 15460
ERROR:2016-02-05 11:33:57,391: Internal Server Error: /api/v1/projects
Traceback (most recent call last):
  File "/home/taiga/.virtualenvs/taiga/lib/python3.4/site-packages/django/core/handlers/base.py", line 164, in get_response
    response = response.render()
  File "/home/taiga/.virtualenvs/taiga/lib/python3.4/site-packages/django/template/response.py", line 158, in render
    self.content = self.rendered_content
  File "/home/taiga/taiga-back/taiga/base/response.py", line 98, in rendered_content
    ret = renderer.render(self.data, media_type, context)
  File "/home/taiga/taiga-back/taiga/base/api/renderers.py", line 632, in render
    context = self.get_context(data, accepted_media_type, renderer_context)
  File "/home/taiga/taiga-back/taiga/base/api/renderers.py", line 609, in get_context
    "post_form": self.get_rendered_html_form(view, "POST", request),
  File "/home/taiga/taiga-back/taiga/base/api/renderers.py", line 492, in get_rendered_html_form
    return form_renderer.render(data, self.accepted_media_type, self.renderer_context)
  File "/home/taiga/taiga-back/taiga/base/api/renderers.py", line 390, in render
    return template.render(context)
  File "/home/taiga/.virtualenvs/taiga/lib/python3.4/site-packages/django/template/backends/django.py", line 74, in render
    return self.template.render(context)
  File "/home/taiga/.virtualenvs/taiga/lib/python3.4/site-packages/django/template/base.py", line 210, in render
    return self._render(context)
  File "/home/taiga/.virtualenvs/taiga/lib/python3.4/site-packages/django/template/base.py", line 202, in _render
    return self.nodelist.render(context)
  File "/home/taiga/.virtualenvs/taiga/lib/python3.4/site-packages/django/template/base.py", line 905, in render
    bit = self.render_node(node, context)
  File "/home/taiga/.virtualenvs/taiga/lib/python3.4/site-packages/django/template/base.py", line 919, in render_node
    return node.render(context)
  File "/home/taiga/.virtualenvs/taiga/lib/python3.4/site-packages/django/template/defaulttags.py", line 224, in render
    nodelist.append(node.render(context))
  File "/home/taiga/.virtualenvs/taiga/lib/python3.4/site-packages/django/template/defaulttags.py", line 329, in render
    return nodelist.render(context)
  File "/home/taiga/.virtualenvs/taiga/lib/python3.4/site-packages/django/template/base.py", line 905, in render
    bit = self.render_node(node, context)
  File "/home/taiga/.virtualenvs/taiga/lib/python3.4/site-packages/django/template/base.py", line 919, in render_node
    return node.render(context)
  File "/home/taiga/.virtualenvs/taiga/lib/python3.4/site-packages/django/template/base.py", line 959, in render
    output = self.filter_expression.resolve(context)
  File "/home/taiga/.virtualenvs/taiga/lib/python3.4/site-packages/django/template/base.py", line 648, in resolve
    obj = self.var.resolve(context)
  File "/home/taiga/.virtualenvs/taiga/lib/python3.4/site-packages/django/template/base.py", line 789, in resolve
    value = self._resolve_lookup(context)
  File "/home/taiga/.virtualenvs/taiga/lib/python3.4/site-packages/django/template/base.py", line 849, in _resolve_lookup
    current = current()
  File "/home/taiga/taiga-back/taiga/base/api/fields.py", line 197, in widget_html
    return self.widget.render(self._name, self._value)
  File "/home/taiga/.virtualenvs/taiga/lib/python3.4/site-packages/django/forms/widgets.py", line 513, in render
    options = self.render_options(choices, [value])
  File "/home/taiga/.virtualenvs/taiga/lib/python3.4/site-packages/django/forms/widgets.py", line 539, in render_options
    for option_value, option_label in chain(self.choices, choices):
  File "/home/taiga/.virtualenvs/taiga/lib/python3.4/site-packages/django/forms/models.py", line 1095, in __iter__
    method = 'all' if self.queryset._prefetch_related_lookups else 'iterator'
AttributeError: 'Manager' object has no attribute '_prefetch_related_lookups'
[05/Feb/2016 11:33:58] "GET /api/v1/projects HTTP/1.1" 500 27
```

**Comment by guglielmo** [comment]
This is related to a regression introduced in django 1.8.6 and it is solved by upgading django version to 1.8.9 in taiga's `requirements.txt`.

**Comment by superalex** [comment]
Thank you very much @guglielmo I'm trying to reproduce the issue but I can't, we use django 1.8.6 in tree.taiga.io. We would like to move to django 1.9 as soon as we can.

Regards!,
